hudi-client/src/test/java/org/apache/hudi/common/HoodieTestDataGenerator.java

```diff
@@ -155,6 +181,19 @@ public static TestRawTripPayload generateRandomValue(
     return new TestRawTripPayload(rec.toString(), key.getRecordKey(), key.getPartitionPath(), TRIP_EXAMPLE_SCHEMA);
   }
 
+  /**
+   * Generates a new avro record with TRIP_UBER_EXAMPLE_SCHEMA, retaining the key if optionally provided.
+   */
+  public TestRawTripPayload generatePayloadForUberSchema(HoodieKey key, String commitTime) throws IOException {
```

Comment:
Please remove Uber from this method name

Reply:
Done